test/factories.rb

```diff
@@ -1,5 +1,10 @@
 # This will guess the User class
-FactoryGirl.define do
+FactoryGirl.define do  factory :user do
```

Comment:
Unnecessary spacing detected.<br>Block body expression is on the same line as the block start.